.rubocop.yml

```diff
@@ -37,4 +37,10 @@ RSpec/InstanceVariable:
   Enabled: false
   
 RSpec/MultipleExpectations:
+  Enabled: false
+  
+RSpec/ExampleLength:
+  Enabled: false
+  
+RSpec/LetSetup:
   Enabled: false
```

Comment:
Missing newline character at the end of the file 

You can configure your IDE to take care of it for you :)